### DISCUSSION TITLE
[Linux] Gothic 1

### DISCUSSION THREAD

**Discussion by Gr33nEyes** [Try/OpenGothic#372] [Q&A]
Возможно ли запустить первую Готику на Линуксе?

**Comment by Try** [comment]
Hi, @Gr33nEyes !

Generally we do not support gothic1. You can still launch it regardless, but don't expect game to work well

**Reply by Gr33nEyes** [reply]
May I fork engine?

**Reply by Try** [reply]
Sure, that is one of the reasons, why engine is public :D 
Also it's ok to submit pull-requests for G1 features
